api/oauth2.js

```diff
@@ -107,6 +117,25 @@ module.exports.load = async function(app, db) {
 
           res.cookie('accountid', userinfo.id);
         }
+        let j4r = newsettings.api.client.j4r.every;
+        let newj4r = {
+                "cpu": 0,
+                "ram": 0,
+                "disk": 0,
+                "servers": 0
+            }
+        if (newsettings.api.client.j4r.enabled == true) {
+            if (guildsinfo.message == '401: Unauthorized') return res.send("Please allow us to know what servers you are in to let the J4R system work properly.")
```

Comment:
Instead of doing this, make this res.send("An error has occurred.") and add a `if (newseetings.api.client.j4r.enabled) if (scopes.replace(/guilds/g, "") == scopes) missingscopes.push("email");` somewhere near https://github.com/real2two/dashactyl/blob/8525187f563ec7c2536d7fbe81759e57bd9d63b1/api/oauth2.js#L57